### ISSUE TITLE
Is this block still needed?

### ISSUE THREAD

**Issue by mattdodge** [nio-blocks/operator#2]
With the addition of the numeric counter, do we still need this block?

**Comment by mattdodge** [comment]
See Numeric Counter (https://github.com/nio-blocks/counter)

**Comment by hansmosh** [comment]
The main difference is that the Operator block has a reset_condition. The NumericCounter has a reset command that should be able to handle the same functionality though.

@cloudformdesign Would you be able to switch over all uses of this block in the lab to the NumericCounter?

**Comment by cloudformdesign** [comment]
yes I can probably change. I will delete this block as soon as the numeric counter is verified.

**Comment by hansmosh** [comment]
@mattdodge It sounds like this can be deleted. Would you like the honor?
